pkg/helpers/helpers_test.go

```diff
@@ -0,0 +1,92 @@
+package helpers
+
+import (
+	"testing"
+
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
+	"github.com/stretchr/testify/require"
+)
+
+func TestDecodeSnowflakeParameterID(t *testing.T) {
```

Comment:
FYI this would be a good place to use table-driven tests. https://dave.cheney.net/2019/05/07/prefer-table-driven-tests. But not that important

Reply:
yeah, I support that, we already have them in few other places